app/services/upload_to_print_house.rb

```diff
@@ -6,5 +9,16 @@ def initialize(csv)
   end
 
   def call
+    upload_file('/Data.in/batch.csv', csv)
+    upload_file('/Trigger.in/trigger.txt', 'trigger')
+  end
+
+  private
+
+  def upload_file(path, contents)
+    Net::SFTP.start(ENV['SFTP_HOST'], ENV['SFTP_USER'], password: ENV['SFTP_PASSWORD']) do |sftp|
+      io = StringIO.new(contents)
```

Comment:
Minor point, but, you don't need the SFTP connection open to create the `StringIO` instance.

```rb
io = StringIO.new(contents)

Net::SFTP.start(ENV['SFTP_HOST'], ENV['SFTP_USER'], password: ENV['SFTP_PASSWORD']) do |sftp|
  sftp.upload!(io, path)
end
```